src/components/Dashboard.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+	<DashboardWidget :items="notes"
+		:loading="loading"
+		:show-more-text="t('notes', 'notes')"
+		:show-more-url="showMoreUrl"
+	>
+		<template v-slot:default="{ item }">
+			<DashboardWidgetItem
+				:target-url="getItemTargetUrl(item)"
+				:main-text="item.title"
+				:sub-text="subtext(item)"
+				:item="item"
+			>
+				<template v-slot:avatar>
+					<div
+						class="note-item"
+						:class="{ 'note-item-favorite': item.favorite }"
+					/>
+				</template>
+			</DashboardWidgetItem>
+		</template>
+		<template v-slot:empty-content>
+			<EmptyContent icon="icon-notes">
+				<template #desc>
+					<p class="notes-empty-content-label">
+						{{ t('notes', 'No notes found') }}
+					</p>
+					<p>
+						<a :href="createNoteUrl" class="button">{{ t('note', 'Create note') }}</a>
+					</p>
+				</template>
+			</EmptyContent>
+		</template>
+	</DashboardWidget>
+</template>
+
+<script>
+import { DashboardWidget, DashboardWidgetItem } from '@nextcloud/vue-dashboard'
+import { EmptyContent } from '@nextcloud/vue'
+import { generateUrl } from '@nextcloud/router'
+import { categoryLabel } from '../NotesService'
+
+export default {
+	name: 'Dashboard',
+
+	components: {
+		DashboardWidget,
+		DashboardWidgetItem,
+		EmptyContent,
+	},
+
+	props: {
+		notes: {
+			type: Array,
+			required: true,
+		},
+	},
+
+	data() {
+		return {
+			loading: false,
+		}
+	},
+
+	computed: {
+		showMoreUrl() {
+			return this.notes.length > 7 ? generateUrl('/apps/notes') : null
+		},
+
+		createNoteUrl() {
+			return generateUrl('/apps/notes/new')
+		},
+
+		getItemTargetUrl() {
+			return (note) => {
+				return generateUrl('/apps/notes/note/' + note.id)
+			}
+		},
+	},
+
+	methods: {
+		subtext(item) {
+			return categoryLabel(item.category)
```

Comment:
Instead of using the category for the subline, which is not _soo_ important (and can also return "No category" in lots of cases, which results in noise), could we use the beginning of the note instead (except the title of course)? :)

That would then be very similar to the Android app main view and give nice context to each note.